ch00python/028dictionaries.ipynb

```diff
@@ -390,33 +349,31 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### No guarantee of order"
+    "### No guarantee of order (before python 3.6)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "\n",
-    "Another consequence of the way dictionaries work is that there's no guaranteed order among the\n",
-    "elements:\n",
+    "Another consequence of the way dictionaries used to work is that there was no guaranteed order among the\n",
+    "elements. However, since Python 3.7, it's guaranteed that dictionaries keep the insertion ordering as it has become a language feature. Read more about [why that changed and how it is still fast](https://stackoverflow.com/a/39980744/1087595).\n",
```

Comment:
Very minor, and I'm not too committed to it -- suggesting a wording which I find slightly clearer, and also removing the "language feature" part (I'm not sure it makes sense unless you know the background of it being an implementation detail before 3.7, and might confuse people)

```suggestion
    "elements. However, since Python 3.7, it's guaranteed that dictionaries return elements in the order in which they were inserted. Read more about [why that changed and how it is still fast](https://stackoverflow.com/a/39980744/1087595).\n",
```